appengine/cloudsql/package.json

```diff
@@ -35,19 +35,20 @@
     "test": {
       "app": {
         "requiredEnvVars": [
+          "SQL_CLIENT",
           "MYSQL_USER",
           "MYSQL_PASSWORD",
           "MYSQL_DATABASE",
-          "YOUR_INSTANCE_CONNECTION_NAME"
+          "POSTGRES_USER",
+          "POSTGRES_PASSWORD",
+          "POSTGRES_DATABASE",
+          "INSTANCE_CONNECTION_NAME"
         ],
         "msg": "Last 10 visits:",
-        "args": ["server.js"]
-      },
-      "deploy": {
```

Comment:
The line below it seems to be MySQL-specific environment variables - but if we use the same environment variables for both clients, then I'll add it back in + update it accordingly.